Libs/Core/ctkJobScheduler.cpp

```diff
@@ -291,7 +307,7 @@ void ctkJobScheduler::stopAllJobs(bool stopPersistentJobs)
       }
 
     job->setStatus(ctkAbstractJob::JobStatus::Stopped);
```

Comment:
Is setStatus thread-safe? It is not reflected in its name. Renaming to `requestSetStatus` would make it more clear.

Reply:
yes it is thread-safe. the job is still on the main thead!! there is no worker associated to it.